consensus/src/user/sortition.rs

```diff
@@ -74,7 +74,6 @@ pub fn create_sortition_hash(cfg: &Config, counter: u32) -> [u8; 32] {
 
     // write input message
     hasher.update(&cfg.seed.inner()[..]);
-    hasher.update(cfg.round.to_le_bytes());
```

Comment:
Shall we remove `round` field from config?

Reply:
No, because it's needed to know the elibigle provisioners

Reply:
Thanks, I'll see if I can remove it next pr